packages/jest/migrations.json

```diff
@@ -19,9 +19,6 @@
     },
     "rename-test-path-pattern": {
       "version": "21.3.0-beta.3",
-      "requires": {
```

Comment:
This was incorrectly requiring Jest v30. Regardless of the Jest version, we renamed the executor option from that version of Nx onwards, so we need the migration to run.